### DISCUSSION TITLE
OpenRct2 doesn't close properly sometimes when exiting the game!

### DISCUSSION THREAD

**Discussion by tylere661** [OpenRCT2/OpenRCT2#22176] [Q&A]
Hello, sometimes when I exit Openrct2, it freezes and then it says program is not responding.  It doesn't affect my gameplay, but it is kinda annoying.  Any ideas why this is happening or is anyone else having this problem?  Also, I haven't seen any changes to the change log lately.

**Comment by 733737** [comment]
This has been going on for around a year. It has to do with freeing images on the image table. It is very difficult to fix. To my knowledge the developers are not working on a fix. This is because it is hard to pinpoint the exact nature of the problem.

**Comment by tylere661** [comment]
Thank you for the information.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
